src/main/kotlin/org/jitsi/nlj/transform/node/AudioRedHandler.kt

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright @ 2019 - present 8x8 Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.nlj.transform.node
+
+import org.jitsi.config.JitsiConfig
+import org.jitsi.metaconfig.config
+import org.jitsi.nlj.PacketInfo
+import org.jitsi.nlj.format.AudioRedPayloadType
+import org.jitsi.nlj.rtp.RedAudioRtpPacket
+import org.jitsi.nlj.rtp.AudioRtpPacket
+import org.jitsi.nlj.rtp.RtpExtensionType
+import org.jitsi.nlj.stats.NodeStatsBlock
+import org.jitsi.nlj.util.BufferPool
+import org.jitsi.nlj.util.ReadOnlyStreamInformationStore
+import org.jitsi.nlj.util.RtpPacketCache
+import org.jitsi.rtp.rtp.RtpPacket
+import org.jitsi.rtp.rtp.header_extensions.AudioLevelHeaderExtension
+import org.jitsi.rtp.util.RtpUtils.Companion.applySequenceNumberDelta
+
+class AudioRedHandler(
+    streamInformationStore: ReadOnlyStreamInformationStore
+) : MultipleOutputTransformerNode("RedHandler") {
+
+    val policy = Config.config.policy
+    val lossThreshold = Config.config.lossThreshold
+    val distance = Config.config.distance
+    val vadOnly = Config.config.vadOnly
+
+    val stats = Stats()
+
+    var audioLevelExtId: Int? = null
+    var redPayloadType: Int? = null
+
+    private val ssrcRedHandlers: MutableMap<Long, SsrcRedHandler> = HashMap()
+
+    // TODO: Hook up to the current packet loss.
```

Comment:
Does this need to be resolved?

Reply:
I don't think we'll end up using it. I should probably remove it from the code.